lib/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   clamp: require("./clamp"),
   combinations: require("./combinations"),
   contain: require("./contain"),
+  decrement: require('./decrement'),
```

Comment:
This is still using single quotes :D Thanks for adapting the other places so fast!